yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -378,7 +386,15 @@ export class Sequencer {
       this.l1ToL2MessageSource.getBlockNumber(),
     ]);
     const min = Math.min(...syncedBlocks);
-    return min >= this.lastPublishedBlock;
+    const [worldState, p2p, l2BlockSource, l1ToL2MessageSource] = syncedBlocks;
+    const result = min >= this.lastPublishedBlock;
+    this.log.debug(`Sync check to last published block ${this.lastPublishedBlock} ${result ? 'succeeded' : 'failed'}`, {
```

Comment:
Doesn't this spam the logs? Even at debug, that would be pretty annoying.

Reply:
A bit, but I'd argue that debug should be annoying, and it's useful for understanding the state of the components that act as dependencies to the sequencer. I'd keep it if possible.